src/app/views/settings/Settings.tsx

```diff
@@ -113,6 +122,10 @@ function Settings(props: ISettingsProps) {
       });
   };
 
+  const handleChangeMode = (permissionModeType: boolean) => {
+    dispatch(changeMode(!permissionModeType));
+  };
+
```

Comment:
https://github.com/LokiLabs/microsoft-graph-explorer-v4/pull/11#discussion_r660880786

Reply:
`Settings.tsx` has serveral other `handleSomeAction` methods that are defined in the file like this. Maybe @ChangshengYan can comment on the best place to put this?

Reply:
I don't have a strong preference on this.. If it's only used in one file then should be fine put to it here. If it'll be shared at some point then we can put it under permission-mode-action-creator.ts